extensions/banner/deployment/src/main/java/io/quarkus/banner/deployment/BannerProcessor.java

```diff
@@ -0,0 +1,49 @@
+package io.quarkus.banner.deployment;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+import io.quarkus.banner.runtime.BannerRecorder;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.annotations.ExecutionTime;
+import io.quarkus.deployment.annotations.Record;
+import io.quarkus.deployment.builditem.HotDeploymentWatchedFileBuildItem;
+import io.quarkus.deployment.util.FileUtil;
+
+public class BannerProcessor {
+
+    @BuildStep
+
```

Comment:
This should probably be removed :)

Reply:
Isn't the `@BuildStep`annoation necessary? I thought this needed as this would run at build time

Reply:
I meant you can remove the extra line 😄

Reply:
oh sure